angularjs/Dialog/appearance-and-styling.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Syncfusion Dialog appearance-and-styling
-description: appearance and styling
+title: Appearance and styling in AngularJS Diagram Control | Syncfusion
```

Comment:
Control name is wrong, check and change.

Reply:
Comment resolved